packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -55,6 +65,7 @@ export class CfnInclude extends core.CfnElement {
   private readonly conditionsScope: core.Construct;
   private readonly resources: { [logicalId: string]: core.CfnResource } = {};
   private readonly parameters: { [logicalId: string]: core.CfnParameter } = {};
+  private readonly parameterValues: { [parameterName: string]: any };
```

Comment:
I would probably still call this `parameters` here as well. Possibly something more descriptive, like `parametersToReplace`. But definitely not `parameterValues`.